spec/features/posts_feature_spec.rb

```diff
@@ -0,0 +1,73 @@
+require 'rails_helper'
+
+feature 'posts' do
+  context 'no posts have been added' do
+    scenario 'should display a prompt to add a post' do
+      visit '/'
+      expect(page).to have_content 'No posts yet!'
+      expect(page).to have_link 'New Post'
+    end
+  end
+end
+
+feature 'Creating posts' do
+  scenario 'can create a job' do
+    visit '/'
+    click_link 'New Post'
+    attach_file('Image', "spec/files/images/maccabees.jpg")
+    fill_in 'Caption', with: 'great gig last night'
+    click_button 'Create Post'
+    expect(page).to have_content('great gig last night')
+    expect(page).to have_css("img[src*='maccabees.jpg']")
+  end
+  it 'needs an image to create a post' do
+    visit '/'
+    click_link 'New Post'
+    fill_in 'Caption', with: "The pen is mightier than the sword"
+    click_button 'Create Post'
+    expect(page).to have_content('Stop, in the name of love! You need to upload an image')
```

Comment:
Metrics/LineLength: Line is too long. [90/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)